package.json

```diff
@@ -14,6 +14,7 @@
     "compass-mixins": "0.12.10",
     "core-js": "3.3.6",
     "css-vars-ponyfill": "2.1.2",
+    "custom-protocol-detection": "github:ismailhabib/custom-protocol-detection",
```

Comment:
Has this been checked out by a PM for licensing?

Reply:
@esethna this was used in the original PR, are we okay here?

Reply:
It's MIT, so we should be fine there

https://github.com/ismailhabib/custom-protocol-detection